app/views/partials/backup_code/_key.slim

```diff
@@ -0,0 +1,11 @@
+.bg-light-blue.bg-personal-key
+  p.bold.center.mt1.pt1.fs-20p
+    = t('users.backup_code.header')
+  .mt4.mb4.px0.sm-px1.py2.center.bg-white.bw2.separator-text.bg-pk-box
+    - code.split('-').each do |word|
+      .inline.fs-20p.bold.navy.monospace(data-personal-key='word')
+        = word
+
+  .center.h5.mt5.mb1.pt1.pb2.fs-12p
+    = t('users.backup_code.generated_on_html',
+        date: content_tag(:strong, I18n.l(Time.zone.today, format: '%B %d, %Y')))
```

Comment:
Does this need to take timezone into account?

Reply:
You inadvertently found the missing generated at timestamp code for the ui.  This partial is actually not being used.